source/config/Config.cpp

```diff
@@ -622,6 +634,14 @@ bool PlainConfig::Jobs::LoadFromCliArgs(const CliArgs &cliArgs)
 
 bool PlainConfig::Jobs::Validate() const
 {
+    if (!handlerDir.empty())
```

Comment:
Do we want to add has_value check over here? Otherwise we might see an segmentation fault for checking is empty on null value.